### PR TITLE
Support royaltyInfo, more stepdefs, cleanup

### PR DESCRIPTION
**Pull request by plaird523**

### CODE REVIEW

File: src/TheLow.sol [hunk 1]
```diff
@@ -193,4 +193,25 @@ constructor(address bigNightAddr) ERC721("partywithray - The Low", "LOW") Owned(
         return uint8(((numerator * precision + denominator - 1) / denominator));
     }
 
+    /// @dev see ERC165
+    function supportsInterface(bytes4 interfaceId) public view virtual override (ERC721) returns (bool) {
+        return interfaceId == 0x2a55205a // ERC2981 -- royaltyInfo
```
Comment (neodaoist) [inline src/TheLow.sol]: Put ERC165 first

File: tests/cucumber.rs [hunk 2]
```diff
@@ -28,14 +28,14 @@ abigen!(TheLow, "out/TheLow.sol/TheLow.json");
 async fn main()
 {
     // Generate bindings for the 3rd party contracts we'll need
-
+    // TODO: Add VRF here if we end up using it
```
Comment (neodaoist) [inline tests/cucumber.rs:31]: I'd prefer we make this a NOTE or Dev note or similar, but not a TODO.

File: tests/unit/TheLow.t.sol [hunk 3]
```diff
@@ -174,4 +174,20 @@ contract TheLowTest is Test {
         low.divideRoundUp(100,0,1);
     }
 
+        function testRoyalty() public {
+            vm.startPrank(team);
+            low.transferFrom(team,alice,2);
```
Comment (neodaoist) [inline tests/unit/TheLow.t.sol:179]: Not blocking and we don't even have CI pipeline setup, but `forge fmt` should be run before opening a PR and for instance that would re-format this line's formatting.

File: tests/unit/TheLow.t.sol [hunk 4]
```diff
@@ -174,4 +174,20 @@ contract TheLowTest is Test {
...
+
+            (address recipient, uint256 amount) = low.royaltyInfo(1, 100_000);
```
Comment (neodaoist) [inline tests/unit/TheLow.t.sol:181]: Nice, love the use of underscores for test readability and maintainability

**Review by neodaoist — CHANGES_REQUESTED** [review]
Very nice. One small change related to `supportsInterface` otherwise LGTM and I just have nits.

File: src/TheLow.sol (hunk 1 above)
Comment (plaird523) [reply src/TheLow.sol]: Fixed

**Review by neodaoist — APPROVED** [review]
LGTM

**Pull request merged**